### PR TITLE
chore: Fix and enable stylelint rule `declaration-property-max-values` for padding

### PR DESCRIPTION
**Pull request by katydecorah**
#### Details

This pull request will set [`declaration-property-max-values`](https://stylelint.io/user-guide/rules/list/declaration-property-value-disallowed-list/) for padding to 1. This will help improve readability and consistency of padding values.

There were a few instances where `0` acted as placeholder, I verified each case to see if that `0` value was required or could be removed when rewritten.

##### Motivation

Part of https://github.com/microsoft/accessibility-insights-web/issues/5187 as an incremental task to enable stylelint rules.

#### Pull request checklist

- [x] Addresses an existing issue: #5187 
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [x] (UI changes only) Added screenshots/GIFs to description above
- [x] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/reports/components/outcome.scss [hunk 1]
```diff
@@ -60,7 +60,8 @@ $outcome-count-border-size: 1.5px;
         line-height: $outcome-chip-icon-size - (2 * $outcome-count-border-size);
         font-size: 11px;
         border-radius: 0 8px 8px 0;
-        padding: 0 6px 0 10px;
+        padding-right: 6px;
```
Comment (katydecorah) [inline src/reports/components/outcome.scss:63]: I verified that `padding-top: 0` and `padding-bottom: 0` are not needed.

File: src/common/components/cards/rule-resources.scss [hunk 2]
```diff
@@ -22,6 +25,7 @@
     }
 
     .rule-details-id {
-        padding: 12px 0;
+        padding-top: 12px;
+        padding-bottom: 12px;
```
Comment (katydecorah) [inline src/common/components/cards/rule-resources.scss:29]: I verified that `padding-right: 0;` and `padding-left: 0` are not needed.

File: src/assessments/assessment-default-message-generator.scss [hunk 3]
```diff
@@ -3,5 +3,7 @@
 .no-failure-view {
     font-size: 15px;
     font-weight: normal;
-    padding: 15px 0 15px 15px;
+    padding-top: 15px;
```
Comment (katydecorah) [inline src/assessments/assessment-default-message-generator.scss:6]: I verified that `padding-right: 0` is not needed.

Comment (shanisebarona) [reply src/assessments/assessment-default-message-generator.scss:6]: I agree!

**Review by shanisebarona — APPROVED** [review]
LGTM! 🚀

**Pull request merged**